prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,40 @@
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
```

Comment:
Yeah, I think I could've made the printing of the current bill into another function because it is repeated and if I was to add extra to this it may have been repeated even more, and if it was to be a larger program some parts of this might end up getting repeated. Thanks for the tip.